### PR TITLE
WIP: refactor(install): improve migration source and permissions

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#623, #553
- Refactor directory creation. encahnce with brace expansion.
- Permit select install from DB to CSV from command line.
- Avoid use chmod 777

### CODE REVIEW

**Comment by ceoaliongroo** [comment]
@shahardo, you can test locally an confirm that's everything works ok?

File: default.install.sh [hunk 1]
```diff
@@ -14,17 +14,17 @@ ADMIN_USERNAME="admin"
...
 bash scripts/build
 
+cd www
+
 # must create the styles/thumbnails/public dir, otherwise the apache will create
 # it with www-data owner an we wouldn't be able to delete it.
-mkdir -p www/sites/default/files/styles/thumbnail/public
-
-cd www
+mkdir -p sites/default/files/{styles,pictures}/thumbnail/public
```
Comment (amitaibu) [inline default.install.sh]: try to remove

File: default.install.sh [hunk 2]
```diff
@@ -14,17 +14,17 @@ ADMIN_USERNAME="admin"
...
+#chmod og+x sites/default
```
Comment (amitaibu) [inline default.install.sh]: remove

File: default.install.sh [hunk 3]
```diff
@@ -34,20 +34,39 @@ drush en devel diff views_ui field_ui migrate_ui mimemail -y
...
 DATA_DIR="../../negawatt-data/sql-migrate-unified"
-drush vset negawatt_migrate_sql 0
-`drush sql-connect` < $DATA_DIR/_negawatt_account_migrate.sql
-`drush sql-connect` < $DATA_DIR/_negawatt_modbus_meter_migrate.sql
-`drush sql-connect` < $DATA_DIR/_negawatt_iec_meter_migrate.sql
-`drush sql-connect` < $DATA_DIR/_negawatt_electricity_migrate.sql
-`drush sql-connect` < $DATA_DIR/_negawatt_electricity_normalized_migrate.sql
-drush mi --all --user=1
-
+FILE_NAME="dump-kiryatgat+astarl+carmiel+eilat.sql.tar.gz"
```
Comment (amitaibu) [inline default.install.sh]: remove hardcoding of file name

File: default.install.sh [hunk 4]
```diff
@@ -34,20 +34,39 @@ drush en devel diff views_ui field_ui migrate_ui mimemail -y
...
+
+if [ -z $1 ]
+then
+  echo Migration using CSV
+else
+  echo Migration using $1
+
+  if [ $1 = "db" ]
+  then
+    UPLOAD_WHOLE_DB=1
```
Comment (amitaibu) [inline default.install.sh]: convert to function.

**Comment by ceoaliongroo** [comment]
Close in favor PR #677

**Pull request closed without merging**